lib/facter/resolvers/aix/networking.rb

```diff
@@ -48,7 +48,7 @@ def populate_with_mtu_and_mac!(interfaces)
               info = line.split("\s")
               interface_name = info[0]
               mac = info[3][/^([0-9a-f]{1,2}[\.:-]){5}([0-9a-f]{1,2})$/]
-              interfaces[interface_name][:mtu] = info[1].to_i
+              interfaces[interface_name][:mtu] = info[1].to_i unless interfaces[interface_name].nil?
               interfaces[interface_name][:mac] = format_mac_address(mac) if mac
```

Comment:
Wondering if we'll have a similar issue if the interface is down, but there is a `mac`? Maybe guard both of these lines with:

```ruby
intf = interfaces[interface_name]
if intf
  intf[:mtu] = ...
  intf[:mac] = ... if mac
end
```